react-native-aztec/src/utils.js

```diff
@@ -0,0 +1,12 @@
+import { Platform } from 'react-native';
+
+// Basically React Native has a mechanism to scale font size automatically.
+// But GM is rendered on top of fragment of which retain flag is true.
+// Hence we deal with fontScale change manually on our side.
+export function getScaledFontSize( fontSize, fontScale ) {
```

Comment:
currently there's no dependency of unit test in `react-native-aztec` but if ok I'd like to install jest and write an unit test for the function🙇

Reply:
Let's hold on a bit while we're still assessing the solution in this PR.

While at it though, what do you have in mind for how such a unit test would be run? Will it become part of the Jest testsuite already in place in gutenberg-mobile?